### ISSUE TITLE
Auto-binded `route.lookup` method?

### ISSUE THREAD

**Issue by oncletom** [delvedor/find-my-way#83]
Hello,

I discover with quite a joy this module :-) It's nice to explain routing without banging the head of people with frameworks.

What do you think of providing an auto-binded `route.lookup` method? It would make it even easier to plug into a `createServer` call.

```js
const {createServer} = require('http');
const router = require('find-my-way')();

createServer(router.lookup).listen(4000);
```

**Comment by mcollina** [comment]
Thanks for reaching out! We still support Node 6, and bind there is slow, and a user of newer Node.js versions can easily bind it themselves.

I think we should not bind it at this point.

**Comment by oncletom** [comment]
Ah, understood. I did not realise binding was still costly in Node 6 😅 

I raised this issue at first because I wrote the following code example:

```js
const {createServer} = require('http');
const router = require('find-my-way')();
createServer(router.lookup.bind(router)).listen(4000);

router.get('/', () => { ... });
```

It did not work because the routes were created after binding the element.
The context of the writing is for a code example, thus I did not want to have to explain the binding nor the declaration order.

I'll try to find a way to make it simple to get :-)
Thanks a bunch for answering and for this nice piece of software 🙂
